lib/tessel/deploy.js

```diff
@@ -35,6 +40,13 @@ function transformKey(value) {
   }, value);
 }
 
+// TODO: Update this with real pre compiled modules url when ready!!
+const BINARY_SERVER_ROOT = 'https://dl.dropboxusercontent.com/u/3531958/tessel-pre-compiled-binary-modules/';
+const BINARY_CACHE_PATH = path.join(Tessel.LOCAL_AUTH_PATH, 'binaries');
```

Comment:
At some point (not this PR) we should rename this var because this path will be used for more than just auth.